packages/@o3r/rules-engine/src/engine/operator/operators/date-based.operators.ts

```diff
@@ -16,6 +16,52 @@ export const inRangeDate: Operator<Date, [DateInput, DateInput], DateInput> = {
   validateRhs: isValidDateRange
 };
 
+
+/**
+ * Check if the value of the variable is in the next x minutes
+ *
+ * @title is in next minutes
+ *
+ * @returns false for dates before `now` and for dates after `now` + `nextMinutes`, true for dates between `now` and `now` + `nextMinutes`
+ */
+export const dateInNextMinutes: Operator<Date, string | number, DateInput> = {
+  name: 'dateInNextMinutes',
+  evaluator: (leftDateInput, minutes, operatorFactValues) => {
+    if (!operatorFactValues) {
+      throw new Error('No operatorFactValues. Unable to retrieve the current time.');
+    }
+    const currentTimeValue = operatorFactValues.o3rCurrentTime as number;
+    return inRangeDate.evaluator(leftDateInput, [currentTimeValue, currentTimeValue + +minutes * 60000]);
+  },
+  factImplicitDependencies: ['o3rCurrentTime'],
+  validateLhs: isValidDateInput,
+  validateRhs: numberValidator
+};
+
+/**
+ * Check if the value of the variable is not in the next x minutes
+ *
+ * @title is not in next minutes
+ *
+ * @returns false for dates before `now` and for dates between `now` and `now` + `nextMinutes`, true for dates after `now` + `nextMinutes`
+ */
+export const dateNotInNextMinutes: Operator<Date, string | number, DateInput> = {
+  name: 'dateNotInNextMinutes',
+  evaluator: (leftDateInput, minutes, operatorFactValues) => {
+    if (!operatorFactValues) {
+      throw new Error('No operatorFactValues. Unable to retrieve the current time.');
+    }
+    const currentTimeValue = operatorFactValues.o3rCurrentTime as number;
+    const now = new Date(currentTimeValue);
+    const leftDate = new Date(leftDateInput);
```

Comment:
why this `new Date`, `leftDateInput` is already a type `Date`

Reply:
`leftDateInput` is of type `type DateInput = Date | string | number;` as it is the case for the oder date-based operators